docs/devops-guide/docker.md

```diff
@@ -9,27 +9,33 @@ sidebar_label: Docker
 In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
 follow these steps:
 
-1. Download and extract the [latest release]
+1. Install git:
+   
+   ```bash
+   apt install git
+   ```
+
+2. Download and extract the [latest release]
     
     Alternatively, to test the latest changes clone the repository:
```

Comment:
If a alternate is mentioned the alternate should work. Either drop the alternate-suggestion or make the alternate work.

Reply:
Sure, but then the command should go on the alternative, not as the first step.